examples/appgw/main.tf

```diff
@@ -1,4 +1,5 @@
-# Create or source the Resource Group.
+### Create or source a Resource Group ###
```

Comment:
I would say we keep single `#` in resource descriptions for this like before, but have `###` for section blocks like "VM-Series" or "Load Balancers" if possible?

Reply:
But this is a **section block** indeed since it has 1 resource, 1 data source and 1 locals block underneath. I wanted to differentiate that from TF Registry reference comments for example, which concern only a single resource or single data source.

Reply:
It seems , that it's similar comment to my https://github.com/PaloAltoNetworks/terraform-azurerm-swfw-modules/pull/24#discussion_r1513544657.

I'm ok with Adrian's approach. Let's do it in that way.